### DISCUSSION TITLE
Follows & Followers: allow to have folders

### DISCUSSION THREAD

**Discussion by rosa2** [mastodon/mastodon#27898] [General]
### Pitch

To have a way to classify followers and following people in folders, like bookmarks in Firefox and other browsers.

### Motivation

I don't remember all the people nicks and spend time to find them.

With this feature I will be able to make conversations with other users in a fast way, facilitating to me and others to create community, to help new people introducing among them in a subject, to make better conversations with collective intelligence and similar things.

Mastodon is going to be much creative and useful than GAFAM. I am completely sure and want to collaborate on this! Already is much better because its federation and many other things. Thanks always! <3

**Comment by ClearlyClaire** [comment]
I feel like lists and personal account notes already cover those purposes.

**Comment by rosa2** [comment]
> I feel like lists and personal account notes already cover those purposes.

Thanks for your feedback.

Please, don't close the issue until I can test and thanks for teaching me, I didn't use those features.
I am a feminist and 25nov is very important for many women life.
I can return after in a sprint as I propose in https://github.com/mastodon/mastodon/issues/16853

Always thanks <3
